build/content-service/Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM sunbird/openjdk-java11-alpine:latest
-RUN apk update \
-    && apk add  unzip \
-    && apk add curl \
-    && adduser -u 1001 -h /home/sunbird/ -D sunbird \
-    && mkdir -p /home/sunbird
-RUN chown -R sunbird:sunbird /home/sunbird
+FROM --platform=linux/x86_64 eclipse-temurin:11.0.20.1_1-jdk-focal
```

Comment:
Why did we change the base image. Did you check the size of the image after these changes. Is it less or more and how much?